internal/flink/command_compute_pool_use.go

```diff
@@ -19,7 +19,6 @@ func (c *command) newComputePoolUseCommand() *cobra.Command {
 		RunE:              c.computePoolUse,
 	}
 
-	pcmd.AddEnvironmentFlag(cmd, c.AuthenticatedCLICommand)
 	pcmd.AddOutputFlag(cmd)
```

Comment:
Please also delete this flag... not sure why it was added since this command doesn't print any tables.
```suggestion
```

Reply:
Done!